libraries/AP_UAVCAN/AP_UAVCAN.h

```diff
@@ -41,6 +42,9 @@
 #define AP_UAVCAN_HW_VERS_MAJOR 1
 #define AP_UAVCAN_HW_VERS_MINOR 0
 
+#define AP_UAVCAN_MAX_LED_DEVICES 4
+#define AP_UAVCAN_LED_DELAY_MILLISECONDS 50
```

Comment:
20Hz seems a little fast? Is that normal?

Reply:
Yes, this was totally normal for my setup